tox.ini

```diff
@@ -62,47 +50,34 @@ commands =
 
 [testenv:unit]
 description = Run unit tests
-deps =
-    pytest
-    coverage[toml]
-    -r{toxinidir}/requirements.txt
-    -r{toxinidir}/tests/unit/requirements.txt
+dependency_groups = dev
 commands =
     coverage run --source={[vars]src_path} \
         -m pytest --ignore={[vars]tst_path}integration -v --tb native -s {posargs}
     coverage report
 
 [testenv:coverage-report]
 description = Create test coverage report
-deps =
-    pytest
-    coverage[toml]
-    -r{toxinidir}/requirements.txt
+dependency_groups = dev
 commands =
     coverage report
 
 [testenv:static]
 description = Run static analysis tests
-deps =
-    bandit[toml]
-    -r{toxinidir}/requirements.txt
+dependency_groups = dev, static
 commands =
     bandit -c {toxinidir}/pyproject.toml -r {[vars]src_path} {[vars]tst_path}
 
 [testenv:integration]
 description = Run integration tests
-deps =
-    -r{toxinidir}/requirements.txt
-    -r{toxinidir}/tests/integration/requirements.txt
+dependency_groups = dev, integration
 commands =
     pytest -v --tb native --ignore={[vars]tst_path}unit --log-cli-level=INFO -s {posargs}
 
 [testenv:src-docs]
```

Comment:
You can remove this section altogether

Reply:
I'll have a look at this on Monday.

Reply:
Indeed what laxydocs generates under src-docs was pretty useless.
I've removed this section here, dependencies, helper script and an item in new PR checklist.